### PR TITLE
Turns shortest angle in OdomChassisController

### PR DESCRIPTION
**Pull request by jazonshou**
### Description of the Change

OdomChassisController will now turn the shortest turn when ``turnToAngle`` is called. 

### Motivation

Currently, turnToAngle will only turn clockwise. For example, if I wanted to turn to 270_deg, it would turn 270 degrees clockwise instead of turning 90 degrees counterclockwise. 

### Possible Drawbacks

None that I can think of right now 

### Verification Process

The ``turnToPoint`` function uses a similar method of constraining the desired turn angle to [-180, 180], and it works, so this change for ``turnToAngle`` should work too. 

### Applicable Issues

#468

### CODE REVIEW

**Comment by Octogonapus** [comment]
Please allow edits from maintainers.

**Comment by jazonshou** [comment]
> Please allow edits from maintainers.

They should be allowed....

**Comment by Octogonapus** [comment]
I get this error when pushing to your branch
```
To github.com:Ryan4253/OkapiLib.git
 ! [remote rejected]   HEAD -> master (permission denied)
error: failed to push some refs to 'github.com:Ryan4253/OkapiLib.git'
```

**Comment by jazonshou** [comment]
> To github.com:Ryan4253/OkapiLib.git

That's odd. I'm trying to merge changes from Yessir120/OkapiLib. Is it possible that there is a problem on my side?

**Comment by Octogonapus** [comment]
I'm not convinced the PR is set up correctly, but regardless, it's now just faster for me to request changes instead

File: include/okapi/api/chassis/controller/chassisController.hpp [hunk 1]
```diff
@@ -57,6 +57,7 @@ class ChassisController {
 
   /**
    * Turns the robot clockwise in place (using closed-loop control).
+   * (turns the shortest angle for OdomChassisControllers)
```
Comment (Octogonapus) [inline include/okapi/api/chassis/controller/chassisController.hpp]: This isn't applicable to all chassis controllers, so it shouldn't be documented here. You can put it [here](https://github.com/OkapiLib/OkapiLib/blob/master/include/okapi/api/chassis/controller/defaultOdomChassisController.hpp) though. Also please rephrase it to be clearer, e.g. "Turns in the direction of the smallest angle"

File: include/okapi/api/chassis/controller/chassisController.hpp [hunk 2]
```diff
@@ -71,6 +72,7 @@ class ChassisController {
 
   /**
    * Sets the target angle for the robot to turn clockwise in place (using closed-loop control).
+   * (turns the shortest angle for OdomChassisControllers)
```
Comment (Octogonapus) [inline include/okapi/api/chassis/controller/chassisController.hpp]: same as above

**Review by Octogonapus — CHANGES_REQUESTED** [review]
Please also add a test. Like this one:
```
diff --git a/test/defaultOdomChassisControllerTest.cpp b/test/defaultOdomChassisControllerTest.cpp
index f4de0276..faf0a8f9 100644
--- a/test/defaultOdomChassisControllerTest.cpp
+++ b/test/defaultOdomChassisControllerTest.cpp
@@ -158,6 +158,11 @@ TEST_F(DefaultOdomChassisControllerTest, TurnAboveThreshold) {
   EXPECT_EQ(controller->lastTurnAngleTargetQAngle, 6_deg);
 }
 
+TEST_F(DefaultOdomChassisControllerTest, BoundTurn) {
+  drive->turnToAngle(270_deg);
+  EXPECT_EQ(controller->lastTurnAngleTargetQAngle, -90_deg);
+}
+
 TEST_F(DefaultOdomChassisControllerTest, SetStateTest) {
   auto stateBefore = drive->getState();
   assertOdomStateEquals(0, 0, 0, stateBefore);
```

**Review by Octogonapus — APPROVED** [review]
